### ISSUE TITLE
Process-Panel div sometimes disappears in Google Chrome

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Illablu/google-refine#102]
```
In Chrome 5.0.375.86 beta on Windows 7, during a reconciling process, the 
process-panel div sometimes edges completely off the top of the browser window 
with only a slim blue line of the bottom edge of the panel remaining. However, 
the process does continue to run and shows the % percentage remaining in the 
browsers title bar.

While researching the issue using Chrome Dev Tools, I happened to uncheck the 
style attribute that was height: and the process-panel div reappeared correctly 
in place.  Not sure how this is being set since there was no CSS attribute for 
height in the element found in source, but the getter seems to be through the 
prototype library itself ? or it's an outright bug in Chrome itself ?

More research needed by other experts here.

Screenshot attached.
```

Original issue reported on code.google.com by `thadguidry` on 26 Jun 2010 at 4:22

Attachments:
- [Fix_for_Chrome_Process_Panel_disappearing.png](https://storage.googleapis.com/google-code-attachments/google-refine/issue-102/comment-0/Fix_for_Chrome_Process_Panel_disappearing.png)

**Comment by GoogleCodeExporter** [comment]
```
Also, interestingly, when the process-panel does appear correctly, the height: 
style parameter does NOT appear in Chrome's Dev Tools.
```

Original comment by `thadguidry` on 26 Jun 2010 at 4:34

**Comment by GoogleCodeExporter** [comment]
Original comment by `dfhu...@gmail.com` on 18 Jul 2010 at 12:18
- Removed labels: **Milestone-1.1**

**Comment by GoogleCodeExporter** [comment]
Original comment by `stefa...@google.com` on 16 Sep 2010 at 9:41
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Seems to have been resolved recently in trunk (with redesign of 2.0) or now 
with Chrome 7.0

```

Original comment by `thadguidry` on 25 Oct 2010 at 2:20

**Comment by GoogleCodeExporter** [comment]
```
Verified as fixed in Version 2.0
```

Original comment by `thadguidry` on 19 Nov 2010 at 8:45
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
Original comment by `tfmorris` on 18 Sep 2012 at 3:01
- Added labels: **Milestone-2.1**
